src/ci/agent.sh

```diff
@@ -39,9 +39,10 @@ cargo fmt -- --check
 # RUSTSEC-2020-0036: a dependency `failure` (pulled from proc-maps) is deprecated
 # RUSTSEC-2019-0036: a dependency `failure` (pulled from proc-maps) has type confusion vulnerability
 # RUSTSEC-2021-0065: a dependency `anymap` is no longer maintained
+# RUSTSEC-2020-0077: `memmap` dependency unmaintained, via `symbolic` (see: `getsentry/symbolic#304`)
 # RUSTSEC-2020-0159: potential segfault in `time`, not yet patched (#1366)
 # RUSTSEC-2020-0071: potential segfault in `chrono`, not yet patched (#1366)
-cargo audit --deny warnings --deny unmaintained --deny unsound --deny yanked --ignore RUSTSEC-2020-0016 --ignore RUSTSEC-2020-0036 --ignore RUSTSEC-2019-0036 --ignore RUSTSEC-2021-0065 --ignore RUSTSEC-2020-0159 --ignore RUSTSEC-2020-0071
+cargo audit --deny warnings --deny unmaintained --deny unsound --deny yanked --ignore RUSTSEC-2020-0016 --ignore RUSTSEC-2020-0036 --ignore RUSTSEC-2019-0036 --ignore RUSTSEC-2021-0065 --ignore RUSTSEC-2020-0159 --ignore RUSTSEC-2020-0071 --ignore RUSTSEC-2020-0077
```

Comment:
`symbolic` depends on `memmap`. We use `memmap2` because `memmap` is deprecated, but `symbolic` has chosen to stick with it.